books/refgram/src/logic/transformations.md

```diff
@@ -99,14 +99,14 @@ equal to the onces the instance had.
 > Assertion given $(c): A cat eats something.\
 > Assertion given $(c): I like __this cat__.
 
-When prefixing a compound, __ze__ will refer the lastest instance of that
+When prefixing a compound, __ze__ will refer to the lastest instance of that
 exact compound. To make sentences shorter, __zei__ can be used before a
 predicate word to refer the latest compound containing this word.
 
 > `ze eberban` will refer to the latest `eberban` instance, while `zei ban` will
 > refer to the latest compound containing `ban`, for exemple `eberban`.
 
-Which is the __latest instance__ is determined by word order in the text, and
+This is the __latest instance__ is determined by word order in the text, and
```

Comment:
```suggestion
Which one is the __latest instance__ is determined by word order in the text, and
```
Sentence with "this" looks nonsensical to me. Word order in the text determined which instance is the latest.